src/components/cards/RelationshipsCard.jsx

```diff
@@ -1,106 +1,510 @@
+              value={currentRole2}
+              options={currentRoles}
+              renderOption={option => (
+                <Text value={option.guid}>{option.label}</Text>
+              )}
+              onChange={(_, newValue) => onChangeRole2(newValue)}
+              getOptionLabel={option => get(option, 'label', '')}
+              getOptionSelected={(option, val) =>
+                option.guid ? option.guid === val : false
+              }
+              renderInput={params => {
+                return (
+                  <TextField
+                    {...params}
+                    style={{ width: 280 }}
+                    variant="standard"
+                    label={intl.formatMessage(
+                      { id: 'SELECT_RELATIONSHIP_ROLE_1' },
+                      {
+                        ind1: individualFirstName
+                          ? individualFirstName
+                          : intl.formatMessage({
+                              id: 'UNNAMED_INDIVIDUAL',
+                            }),
+                      },
+                    )}
+                  />
+                );
+              }}
+            />
+          </div>,
+          <div
+            id="roles2-div"
+            style={{ width: 'fit-content', minWidth: 470 }}
+          >
+            <Autocomplete
+              id="roles2"
+              clearOnEscape
+              style={{ marginTop: 12 }}
+              value={currentRole1}
+              options={currentRoles}
+              renderOption={option => (
+                <Text value={option.guid}>{option.label}</Text>
+              )}
+              onChange={(_, newValue) => onChangeRole1(newValue)}
+              getOptionLabel={option => get(option, 'label', '')}
+              getOptionSelected={(option, val) =>
+                option.guid ? option.guid === val : false
+              }
+              renderInput={params => {
+                return (
+                  <TextField
+                    {...params}
+                    style={{ width: 280 }}
+                    variant="standard"
+                    label={intl.formatMessage(
+                      { id: 'SELECT_RELATIONSHIP_ROLE_2' },
+                      {
+                        ind2: selectedIndividualFirstName,
+                      },
+                    )}
+                  />
+                );
+              }}
+            />
+          </div>,
+        ]}
+      </DialogContent>
+      <DialogActions
+        style={{
+          padding: '0px 24px 24px 24px',
+          display: 'flex',
+          alignItems: 'flex-end',
+          flexDirection: 'column',
+        }}
+      >
+        {postRelationshipError && (
+          <CustomAlert
+            style={{ margin: '20px 0', width: '100%' }}
+            severity="error"
+            titleId="SERVER_ERROR"
+          >
+            <Text key={postRelationshipError} variant="body2">
```

Comment:
you shouldn't need a key here.

Reply:
Also a vestige of things removed. Good catch. Done.